@ornikar/jest-config-react-native/__mocks__/@storybook/react-native.jsx

```diff
@@ -83,7 +83,12 @@ exports.storiesOf = (groupName) => {
             : ({ children }) => decorateStory(() => children, [...localDecorators, ...globalDecorators])(context);
 
           const rtlApi = render(story(context), { wrapper: WrappingComponent });
-          if (waitForExpectation) await waitFor(() => waitForExpectation(rtlApi, expect, { parameters }));
+          if (waitForExpectation) {
+            await waitFor(async () => {
```

Comment:
this code is exactly the same, no ?

Reply:
Yes it is sadly :( I've been digging a bit for the stories performance on `learner-apps` and thought it may impact the `waitFor` in our stories, with no avail